src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,344 @@
+@import "../../patternfly-utilities";
+
+.pf-c-sequence {
+  // Sequence Control
+  --pf-c-sequence__control--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderBottomWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderTopColor: transparent;
+  --pf-c-sequence__control--BorderBottomColor: transparent;
+  --pf-c-sequence__control--hover--BorderTopWidth: var(--pf-global--BorderWidth--lg);
```

Comment:
these should be organized by their state not their property, I think it makes it easier to scan that way.

Reply:
just rearranged that, let me know if theres anything else I should rearrange to make it easier to scan

Reply:
👍